tests/ui/feature-gates/feature-gate-contracts.stderr

```diff
@@ -18,26 +18,6 @@ LL | #[core::contracts::ensures(|ret| *ret > 0)]
    = help: add `#![feature(contracts)]` to the crate attributes to enable
    = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date
 
-error[E0658]: contracts are incomplete
```

Comment:
This was redundant. We're already issuing a library feature error above, since we're *just* using a library feature. 

Changing it to supporting issuing two feature gates is an undue burden to the parser impl unless we want to gate the internal machinery under both `contracts` and `contracts_internals`.